src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition-duration: 0.3s;
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/224971033-a0961484-fe1e-4bb7-8b01-b29dcc85c1d2.png)

Reply:
and move 0.3s to a variable

Reply:
Hi. Thank you. I fixed and add variable.